docs/platform/howto/create_new_service.rst

```diff
@@ -1,60 +1,30 @@
 ﻿Create a new service
 ====================
 
-You can add services for the various Aiven products in the Aiven web console.
+Follow these steps to create a new service in the `Aiven Console <https://console.aiven.io/>`_.  You can also :ref:`create a service using the Aiven CLI <avn-cli-service-create>`.
 
-To create a new service:
+#. In the project you want to create a service in, go to **Services**.
 
-1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+#. On the **Current services** page, click **Create service**.
 
-2. On the *Services* page, click **Create a new service**.
+#. Select the service that you want to create.
 
-   This opens a new page with the available service options.
+#. Select the cloud provider and region that you want to run your service on.
 
-   .. image:: /images/platform/concepts/console_create_service.png
-      :alt: Aiven Console view for creating a new service
+   .. note:: 
+	   The pricing for the same service may vary between different providers and regions. The service summary shows you the pricing for your selected options.
 
-3. Select the main properties for your service:
+#. Select a service plan. This determines the number of servers and what kind of memory, CPU, and disk resources are allocated to your service.
 
-   a. Select the service type.
+#. Add :doc:`disk storage <../concepts/dynamic-disk-sizing>` for your service by using the slider, if needed. The cost for the additional storage in the service summary section.  
```

Comment:
```suggestion
#. Add :doc:`disk storage <../concepts/dynamic-disk-sizing>` for your service using the slider, if needed. You can view the cost for the additional storage in the service summary section.
```

Reply:
Keeping the original text, I think this line, "The cost for the additional storage in the service summary section." doesn't quite make sense and is missing a verb (subject-verb agreement).